samples/demos/end_to_end_tank/README.md

```diff
@@ -0,0 +1,87 @@
+End to End Tank Monitoring IoT Sample Application
+=================================================
+
+This example is part of the Digi's End-to-End Tank Monitoring IoT demo. It
+demonstrates how to use the XBee 3 Cellular modules to communicate with Digi
+Remote Manager and use the BLE interface to talk to mobile apps applied to
+the tank monitoring vertical.
+
+In theory, the XBee 3 Cellular module this example runs in corresponds to a
+tank controller and is connected to the following peripherals but, for
+demonstration purposes, they are emulated:
```

Comment:
In theory, the XBee 3 Cellular module from this example corresponds to a
tank controller and it is connected to the following peripherals but, for
demonstration purposes, they are all emulated